app/reducers/channels.js

```diff
@@ -386,6 +386,17 @@ const filtersSelector = state => state.channels.filters
 const filterSelector = state => state.channels.filter
 const nodesSelector = state => state.network.nodes
 
+const channelMatchesQuery = (channel, nodes, searchQuery) => {
+  const node = nodes.find(n => channel.remote_pubkey === n.pub_key)
+  const query = searchQuery.toLowerCase()
+
+  const remoteNodePub = (channel.remote_node_pub || '').toLowerCase()
+  const remotePubkey = (channel.remote_pubkey || '').toLowerCase()
+  const displayName = (node ? node.alias : '' || '').toLowerCase()
```

Comment:
nit: could call `.toLowerCase` on the values themselves

Reply:
I'm unsure what you mean. The values may be undefined in which case calling `toLowerCase` on it may result in an error. The approach taken here ensures that `toLowerCase` is called on a string regardless of wether the value was set or not.

Reply:
Roger, no need to change.